src/blog/2024/02/using_webcam_with_node-red.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using a webcam with Node-RED
+subtitle: Guide to integrating webcam features into Node-RED applications.
+description: Guide to integrating webcam features into Node-RED applications.
+date: 
+authors: ["sumit-shinde"]
+image: /blog/2024/02/images/using_webcam_with_node-red_capturing_images_by_passing_payload.gif
```

Comment:
@Yndira-FlowForge Can you make a better tile and have Sumit upload it? (He's using a fork, making it hard for you to push it directly)
```suggestion
image: /blog/2024/02/images/using_webcam_with_node-red_capturing_images_by_passing_payload.gif
```

Reply:
@ZJvandeWeg @sumitshinde-84 here it is
![Using-a-webcam-with-Node-RED](https://github.com/FlowFuse/website/assets/129537638/c7bec9f2-cbb6-4f37-93b0-a3d1bd2a7751)